runtime/src/account_info.rs

```diff
@@ -16,7 +16,7 @@ pub type Offset = usize;
 pub type StoredSize = u32;
 
 /// specify where account data is located
-#[derive(Debug, PartialEq)]
+#[derive(Debug, PartialEq, Eq)]
```

Comment:
This doesn't seem related to the other changes for removing the type annotation.